### DISCUSSION TITLE
Actions: Removing Node12, other malarkey

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#1160] [General]
**Node12 Deprecation:** Many of the actions give warnings about using Node12, since it is now deprecated and will stop working in a year or so. I believe that all the warnings in PRQL are now generated by external packages.

As a result of an [issue I raised,](https://github.com/jetli/wasm-bindgen-action/issues/25) these packages have been updated to use Node16. We should ensure that our GHA's are updated to use them.

* jetli/wasm-bindgen-action@0.2.0 and
* jetli/wasm/wasm-pack-action@0.4.0

I also created an issue for [actions-rs](https://github.com/actions-rs/cargo/issues/222) and [baptiste0928/cargo-install](https://github.com/baptiste0928/cargo-install/issues/8) but no response.

**Other Issues:** There are also a bunch of error messages about Environment Variables that I don't understand.

**Comment by max-sixty** [comment]
Dependabot auto-upgrades actions so we'll get those as PRs when they come out.

Thanks for the upstream PRs
